pkg/skaffold/debug/transform.go

```diff
@@ -120,18 +159,58 @@ func transformPodSpec(metadata *metav1.ObjectMeta, podSpec *v1.PodSpec, retrieve
 	portAlloc := func(desiredPort int32) int32 {
 		return allocatePort(podSpec, desiredPort)
 	}
-	// containers are required to have unique name within a pod
+	// map of containers -> debugging configuration maps; k8s ensures that a pod's containers are uniquely named
 	configurations := make(map[string]map[string]interface{})
+	// the container images that require debugging support files
+	var containersRequiringSupport []*v1.Container
+	// the set of image IDs required to provide debugging support files
+	requiredSupportImages := make(map[string]bool)
```

Comment:
I think we only set each value of the dictionary to true so maybe we should use a set instead?

Reply:
If only Go had a set!  😱